DataFormats/PatCandidates/src/PackedTriggerPrescales.cc

```diff
@@ -7,13 +7,13 @@ pat::PackedTriggerPrescales::PackedTriggerPrescales(const edm::Handle<edm::Trigg
   prescaleValues_.resize(handle->size(), 0);
 }
 
-int pat::PackedTriggerPrescales::getPrescaleForIndex(int index) const {
+double pat::PackedTriggerPrescales::getPrescaleForIndex(int index) const {
   if (unsigned(index) >= triggerResults().size())
     throw cms::Exception("InvalidReference", "Index out of bounds");
   return prescaleValues_[index];
 }
 
-int pat::PackedTriggerPrescales::getPrescaleForName(const std::string &name, bool prefixOnly) const {
+double pat::PackedTriggerPrescales::getPrescaleForName(const std::string &name, bool prefixOnly) const {
```

Comment:
do we wish to consider protecting this against people containing to call this expecting an int via the template with default to int and static assert for having an double/FracPrescale